plugins/languages/rustaceanvim.nix

```
@@ -27,92 +27,127 @@ in
       example = null;
     };
 
-    tools = {
-      executor = helpers.defaultNullOpts.mkNullable (
-        with helpers.nixvimTypes;
-        either strLuaFn (enum [
+    tools =
+      let
+        executors = [
           "termopen"
           "quickfix"
           "toggleterm"
           "vimux"
-        ])
-      ) "termopen" "How to execute terminal commands.";
+          "neotest"
+        ];
+        test_executors = executors ++ [ "background" ];
```

```suggestion
        testExecutors = executors ++ [ "background" ];
```
In general, we use camelCase when coding in nix.

---

I see, the rustaceanvim snake_case must have thrown me off..